ipn/ipnserver/server.go

```diff
@@ -320,6 +321,12 @@ func isReadonlyConn(ci connIdentity, operatorUID string, logf logger.Logf) bool
 		// has a different last-user-wins auth model.
 		return false
 	}
+	// TODO(maisem): what should we do here? This is the easiest option to allow
```

Comment:
tailscaled should authenticate that the connection came from the web/CGI script, and the web/CGI shoudl authenticate the client HTTP request came came from a Synology admin using Synology's cookie auth stuff.

Reply:
Right now we don't restrict access to Synology admins. I know some Synology apps do that, but I have no idea how to configure that. Will dig into this.

Reply:
Minor digging suggests that we can actually not restrict web access to only admins https://old.reddit.com/r/synology/comments/8bvjkz/application_permissions/dxadkvv/

Reply:
Can we not block getting the next version of the synology package delivered to synology for review?

Reply:
@bradfitz thanks for the suggestion to use `/etc/group`, I think this works.